tests/test_endpoint_buffers.py

```diff
@@ -38,7 +42,217 @@ def forward_setup_fixture():
     ConstSettings.ForwardMarketSettings.ENABLE_FORWARD_MARKETS = original_enable_forward_markets
 
 
+@pytest.fixture(name="general_setup")
+def general_setup_fixture():
+    """Create area with spot market"""
```

Comment:
I like to link a tip from Spyros here: https://github.com/gridsingularity/gsy-e/pull/1568#discussion_r1043071363